x-pack/plugins/apm/common/utils/formatters/duration.ts

```diff
@@ -46,7 +47,8 @@ export function getUnitLabelAndConvertedValue(
           defaultMessage: 'h',
         }),
         convertedValue: asDecimalOrInteger(
-          moment.duration(value / 1000).asHours()
+          moment.duration(value / 1000).asHours(),
```

Comment:
Can you extract this calculation and re-use it for all units, to make it obvious that we are converting from milliseconds.

```ts
const ms = value / 1000
```

```suggestion
          moment.duration(ms).asHours(),
```